### PR TITLE
Switch to Stripe package instead of script

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
Noticed the TODO so figured might as well.

### Dragons

Not sure if `await` in `handleClick` is kosher but seems to work. Also not sure if we want to log out the errors to sentry.

### How Has This Been Tested?

https://github.com/AudiusProject/audius-client/assets/3893871/3df016f2-2c25-45bd-b832-48267349b10d

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx [hunk 1]
```diff
@@ -73,7 +70,15 @@ export const StripeBuyAudioButton = () => {
         amount,
         destinationWallet: (await getRootSolanaAccount()).publicKey.toString()
       })
-      const stripeOnRampInstance = StripeOnRamp(STRIPE_PUBLISHABLE_KEY)
+      if (!STRIPE_PUBLISHABLE_KEY) {
+        throw new Error('Stripe publishable key not found')
+      }
+      const stripeOnRampInstance = await loadStripeOnramp(
```
Comment (schottra) [inline packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx:76]: Function was already `async` and using an `await` so I wouldn't sweat it.

Just worth noting that event handlers aren't going to block on async functions. If I understand correctly, most of this code is going to run after the click handler has had a chance to finish it's capture/bubble cycle and return back to React.

**Review by schottra — APPROVED** [review]
Good idea.
Probably needs a decent amount of testing since we don't know how much changed between the script and the official package.
Also probably want a client team sign off on this.

**Comment by schottra** [comment]
@dharit-tan For your second question: Yes, we should definitely log out errors so we can catch any issues when we deploy it.

**Comment by dharit-tan** [comment]
cc @dylanjeffers to check out `package-lock.json` changes.

**Review by rickyrombo — APPROVED** [review]

**Comment by rickyrombo** [comment]
> @dharit-tan For your second question: Yes, we should definitely log out errors so we can catch any issues when we deploy it.

Good news is any `console.error`s make it to Sentry

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rt-stripe

**Review by dylanjeffers — APPROVED** [review]
updated locks for ya

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rt-stripe

File: packages/web/package.json [hunk 2]
```diff
@@ -83,6 +83,8 @@
     "@sentry/integrations": "6.16.1",
     "@solana/spl-token": "0.1.8",
     "@solana/web3.js": "1.53.0",
+    "@stripe/crypto": "0.0.4",
+    "@stripe/stripe-js": "^1.54.1",
```
[the remark this answers is not in the document]
Comment (dharit-tan) [reply packages/web/package.json]: @rickyrombo as in should or should not pin?

[the remark this answers is not in the document]
Comment (rickyrombo) [reply packages/web/package.json]: pin it!

[the remark this answers is not in the document]
Comment (rickyrombo) [reply packages/web/package.json:87]: @dharit-tan hmm these aren't pinned in main, can you pin?

**Pull request merged**